dockerfiles/alpine_3.9

```diff
@@ -144,13 +142,13 @@ COPY files/luarocks-config.lua /usr/local/etc/tarantool/rocks/config-5.1.lua
 
 ARG ROCKS_INSTALLER
 RUN set -x \
-    && apk add --no-cache --virtual .run-deps \
-        mariadb-client-libs \
+    && apk add --no-cache \
+        mariadb-client \
```

Comment:
The package on Alpine 3.5 [offers](https://pkgs.alpinelinux.org/contents?branch=v3.7&name=mariadb-client-libs&arch=x86_64&repo=main) those files:

- /usr/lib/libmysqlclient.so.18.0.0
- /usr/lib/libmysqlclient.so.18

So, you need a mysqlclient dynamic library. Okay. But the `mariadb-client` package on Alpine 3.9 [does not offer](https://pkgs.alpinelinux.org/contents?file=&path=&name=mariadb-client&branch=v3.9&repo=main&arch=x86_64) it.

It is [provided](https://pkgs.alpinelinux.org/contents?file=libmysqlclient.so*&path=&name=&branch=v3.9&repo=main&arch=x86_64) by the `mariadb-connector-c-dev` package.

How do you test your change?

Reply:
Right, corrected.

Reply:
Let's continue in PR #196.